packages/onboarding/src/step-container/index.tsx

```diff
@@ -0,0 +1,231 @@
+import { ActionButtons } from '@automattic/onboarding';
+import classNames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { ReactChild, ReactElement } from 'react';
+import StepNavigationLink from '../step-navigation-link';
+import './style.scss';
+
+interface Props {
+	headerText: string | ReactChild;
+	subHeaderText: string | ReactChild;
+	stepContent: ReactElement;
+	hideBack?: boolean;
+	hideSkip?: boolean;
+	hideNext?: boolean;
+	formattedHeader?: ReactElement;
+	shouldHideNavButtons?: boolean;
+	skipButtonAlign?: 'top' | 'bottom';
+	skipHeadingText?: string;
+	hideFormattedHeader?: boolean;
+	headerImageUrl?: string;
+	backLabelText?: string;
+	skipLabelText?: string;
+	nextLabelText?: string;
+	align: 'left' | 'center';
+	className?: string;
+	// Displays an <hr> above the skip button and adds more white space
+	isLargeSkipLayout?: boolean;
+	isExternalBackUrl?: boolean;
+	headerButton?: ReactElement;
+	customizedActionButtons?: ReactElement;
+	isWideLayout?: boolean;
+	isFullLayout?: boolean;
+	isHorizontalLayout?: boolean;
+	goBack?: () => void;
+	goNext?: () => void;
+	flowName?: string;
+	intent?: string;
+	stepName?: string;
+	stepSectionName?: string;
+	recordTracksEvent: () => void;
+}
+
+const StepContainer: React.FC< Props > = ( {
+	headerText,
+	subHeaderText,
+	stepContent,
+	hideBack,
+	hideSkip,
+	hideNext = true,
+	formattedHeader,
+	shouldHideNavButtons,
+	skipButtonAlign = 'bottom',
+	skipHeadingText,
+	headerImageUrl,
+	hideFormattedHeader,
+	backLabelText,
+	skipLabelText,
+	nextLabelText,
+	align,
```

Comment:
You're right, `align` isn't currently used. It's still there because it was used in the `FormattedHeader` component, that I had to pass to `StepContainer` because of Calypso dependencies. I'll remove it for now.